client/src/components/tailwind/TailwindChatBox.tsx

```diff
@@ -1,263 +0,0 @@
-import React, { useEffect, useRef, useState } from 'react';
```

Comment:
MessageBubble didn't cover all the error case handling on this component, it didn't handle the `overloaded` error